src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div id="Calcu">
+      <div className="rowA Display">011</div>
+      <div className="rowB">
+        <button type="button" className="Calc-button">AC</button>
+        <button type="button" className="Calc-button">+/-</button>
+        <button type="button" className="Calc-button">%</button>
+        <button type="button" className="Calc-button Calcu-operator">&#247;</button>
+      </div>
+      <div className="rowC">
+        <button type="button" className="Calc-button">7</button>
+        <button type="button" className="Calc-button">8</button>
+        <button type="button" className="Calc-button">9</button>
+        <button type="button" className="Calc-button Calcu-operator">&#215;</button>
+      </div>
+      <div className="rowD">
+        <button type="button" className="Calc-button">4</button>
+        <button type="button" className="Calc-button">5</button>
+        <button type="button" className="Calc-button">6</button>
+        <button type="button" className="Calc-button Calcu-operator">-</button>
+      </div>
+      <div className="rowE">
+        <button type="button" className="Calc-button">1</button>
+        <button type="button" className="Calc-button">2</button>
+        <button type="button" className="Calc-button">3</button>
+        <button type="button" className="Calc-button Calcu-operator">+</button>
+      </div>
+      <div className="rowF">
+        <button type="button" className="Calc-button-zero">0</button>
+        <button type="button" className="Calc-button">.</button>
+        <button type="button" className="Calc-button Calcu-operator">=</button>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] 👍 Great job adding the `Calculator` component! However, I noticed that you're using the older functional component syntax with the `function` keyword. I recommend using arrow functions, which represent the latest techniques in React. This is important for keeping your code up-to-date and aligned with modern practices. You can refactor your `Calculator` component like this:

```jsx
import React from 'react';

const Calculator = () => {
  // Your component code here
};

export default Calculator;
```

By using arrow functions for your functional components, you'll ensure that your codebase follows current best practices in the React ecosystem. Keep up the excellent work! 👍🚀